lib/child_process.js

```diff
@@ -491,12 +496,12 @@ function execFileSync(/*command, args, options*/) {
 exports.execFileSync = execFileSync;
 
 
-function execSync(/*command, options*/) {
+function execSync(command /*, options*/) {
   var opts = normalizeExecArgs.apply(null, arguments);
   var inheritStderr = opts.options ? !opts.options.stdio : true;
 
-  var ret = spawnSync(opts.file, opts.args, opts.options);
-  ret.cmd = opts.cmd;
+  var ret = spawnSync(opts.file, opts.options);
+  ret.cmd = command;
```

Comment:
I suppose the change on this line is harmless but I don't quite see why you made it.  Because the format of `opts.cmd` may change in the future?